pdc/apps/compose/serializers.py

```diff
@@ -113,10 +113,14 @@ class ComposeTreeVariantField(serializers.Field):
 
     def to_internal_value(self, data):
         request_data = self.context.get("request").data
-        compose = None
-        if request_data:
-            compose = request_data.get("compose")
-        variant = Variant.objects.get(compose__compose_id=compose, variant_uid=data)
+        variant_all = Variant.objects.all()
+        if data not in [tmp['variant_uid'] for tmp in variant_all.values()]:
```

Comment:
@simozhan , I think this step can been removed, we don't need to verify the variant whether exist in DB or unrelated compose.